mozci/push.py

```diff
@@ -1030,3 +1030,72 @@ def make_push_objects(**kwargs):
             cur._child = pushes[i + 1]
 
     return pushes
+
+
+def make_summary_objects(**kwargs):
```

Comment:
I know you were copying `make_push_objects` here, but would you mind not using `**kwargs` and including a docstring describing the parameters? Same for the two below. I'd like to fix up the `make_push_objects` function to do this as well.

Reply:
Sure, I will make the change in an upcoming patch; in that I will use positional/named arguments. 

Also, generally speaking, more docstrings - I forgot about them this time for some reason.